onchain/rollups/contracts/consensus/quorum/IQuorumFactory.sol

```diff
@@ -0,0 +1,61 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {Quorum} from "./Quorum.sol";
+import {IHistory} from "../../history/IHistory.sol";
+
+/// @title Quorum factory interface
+interface IQuorumFactory {
+    // Events
+
+    /// @notice A new quorum was deployed.
+    /// @param _validators The initial set of quorum validators
+    /// @param _quorum The quorum
+    /// @dev MUST be triggered on a successful call to `newQuorum`.
+    event QuorumCreated(address[] _validators, Quorum _quorum);
+
+    // Permissionless functions
+
+    /// @notice Deploy a new quorum.
+    /// @param _validators The initial set of quorum validators
+    /// @param _shares The initial set of quorum shares
```

Comment:
This is not really a set of shares.
Maybe we should say `The initial distribution of shares amongst validators`.
This applies to other parts of this file as well.